Explorer/Assets/DCL/Ipfs/ServerAbout.cs

```diff
@@ -8,7 +8,6 @@ public class ServerAbout
         public ServerConfiguration configurations;
         public ContentEndpoint content;
         public ContentEndpoint lambdas;
-
-        // public CommsConfig? comms; // TODO for comms
+        public CommsInfo comms;
```

Comment:
I think it should be nullable: `CommsInfo?` since sometimes is not contained in the response. Check if serialization keeps working, otherwise ignore

Reply:
good point!